src/hist/basehist.py

```diff
@@ -312,6 +316,7 @@ def plot1d(
         self,
         *,
         ax: "Optional[matplotlib.axes.Axes]" = None,
+        overlay: "Optional[str]" = None,
```

Comment:
```suggestion
        overlay: "Optional[Union[str, int]]" = None,
```

Reply:
I'm not really sure why there are so many quotes on types, but we'll drop Python 3.6 after the next release anyway, so the quotes will go away then.